src/tailwind.src.css

```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400|Open+Sans:400,500,700');
+
+@tailwind preflight;
+@tailwind components;
+
+@import './components/Button/Button.css';
```

Comment:
I couldn't get that to work. It looks like we could use a postcss plugin to do that if we want, but since it's not standard css, it's not supported out of the box.